.github/workflows/flake.yml

```diff
@@ -0,0 +1,19 @@
+name: update-flake-lock
+on:
+  workflow_dispatch: # allows manual triggering
+  schedule:
+    - cron: '0 0 * * 0' # runs weekly on Sunday at 00:00
```

Comment:
If this is something you want to incorporate, let me know if you want to change the default.